content-src/components/DiscoveryStreamComponents/DSEmptyState/DSEmptyState.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+export class DSEmptyState extends React.PureComponent {
+  render() {
+    return (
+      <div className="section-empty-state">
+        <div className="empty-state-message">
+          <h2>You are caught up!</h2>
+          <p>Check back later for more stories.</p>
```

Comment:
Are we intentionally not exposing these strings to localization?

Reply:
Yes, for 68 keeping the strings to en locale only.  I believe 69+ we do need to localize.